docs/usage/docker-shim-removal.md

```diff
@@ -0,0 +1,47 @@
+# Kubernetes dockershim removal
+
+## What's happening?
+With Kubernetes v1.20 the built-in dockershim [was deprecated](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#dockershim-deprecation) and is scheduled to be [removed with v1.24](https://github.com/kubernetes/enhancements/issues/2221).
+Don't Panic! The Kubernetes community has [published a blogpost](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/) and an [FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/) with more information.
+
+Gardener also needs to switch from using the built-in dockershim to `containerd`.
+Gardener will not change running Shoot clusters. But changes to the container runtime will be coupled to the K8s version selected by the Shoot:
+- starting with K8s version 1.22 Shoots not explicitly selecting a container runtime will get `containerd` instead of `docker`. Shoots can still select `docker` explicitly if needed.
+- starting with K8s version 1.23 `docker` can no longer be selected. 
+
+At this point in time, we have no plans to support other container runtimes, such as `cri-o`.
+    
+## What should I do?
+As a gardener operator:
+- add `containerd` and `docker` to `.spec.machineImages[].versions[].cri.name` in your CloudProfile to allow users selecting a container runtime for their Shoots (see below). **Note:** [not all operating systems support all container runtimes](#differences-in-operating-systems)
+- update your cloud provider extensions to avoid a node rollout when a Shoot is configured from `cri=nil` to `cri.name: docker`:
```

Comment:
We need similar information also for the os-extensions.

Reply:
Good catch! I'll update the table below with this information. I'll also move the information about extensions such that the details are all in one place.